content/docs/command-reference/run.md

```diff
@@ -73,23 +73,34 @@ $ dvc run -n printer -d write.sh -o pages ./write.sh
 $ dvc run -n scanner -d read.sh -d pages -o signed.pdf ./read.sh
 ```
 
+Stage dependencies can be any file in the workspace, either untracked, or much
```

Comment:
mention external deps?

Reply:
Good point. Maybe I'll just not mention the workspace to avoid complicating the paragraph... ⌛

Reply:
OK removed worspace mention from here and added a bullet about external deps/outs in cb80e026.